wwwbase/search.php

```diff
@@ -236,6 +263,21 @@
 }
 
 $results = SearchResult::mapDefinitionArray($definitions);
+
+if(SPOOF_ENABLED && $cuv_spoofed) {
+
+  function replaceSpoofedWord($definition, $cuv) {
+    $pattern = '/<b>(.*)<\/b>(\s.*)/';
+    $replacement = sprintf('<b>%s</b>${2}', mb_strtoupper($cuv));
+    $definition->htmlRep = preg_replace($pattern, $replacement, $definition->htmlRep);
```

Comment:
...and a `, 1` last argument here to ensure that only the first `<b>` tag gets replaced.